CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [8.1.0]
+### Added
+- Fix: saving serialized keyring for which corresponding keyring class is not present ([#169](https://github.com/MetaMask/KeyringController/pull/169))
```

Comment:
```suggestion
- Fix saving serialized keyring for which corresponding keyring class is not present ([#169](https://github.com/MetaMask/KeyringController/pull/169))
```
small update: removed `:` to match the previous log format

Reply:
That is auto generated actually. Not sure if I should change.

Reply:
I updated the changelog.